generic3g/specs/StateItemSpec.F90

```diff
@@ -187,6 +198,13 @@ function I_make_adapters(this, goal_spec, rc) result(adapters)
          integer, optional, intent(out) :: rc
       end function I_make_adapters
 
+      function I_get_aspect_priorities(src_spec, dst_spec) result(aspect_order)
+         import StateItemSpec
+         character(:), allocatable :: order
+         class(StateItemSpec), intent(in) :: src_spec
+         class(StateItemSpec), intent(in) :: dst_spec
+      end function I_get_aspect_priorities
+
```

Comment:
Is this still needed?

Reply:
Yes - as per comment above.    Note - that I would love a better name.    Basically I wan the base method to call the subclass method (to avoid tedium of creating a long vector manually in the subclasses).   So the two procedures do  the same thing except that they return the information in a different form (string vector vs colon-separated string).

Or make them subroutines and then we could overload the same name.   Suggestions welcome.